src/components/WhaleSightingViewer/WhaleSightingViewer.tsx

```diff
@@ -1,62 +1,62 @@
+import { useEffect, useState } from "react";
+import SightingList from "./SightingList";
+import { fetchAllApprovedSightings, WhaleSighting, getPendingSightings } from "../../clients/apiClient";
 import { Login } from "../Login/Login";
 import "./WhaleSightingViewer.scss";
 import { WhaleSightingFilters } from "./WhaleSightingFilters";
-import { useState } from "react";
 
 interface WhaleSightingViewerProps {
-	loggedIn: boolean;
+	loggedIn: boolean,
+	isAdminPage: boolean;
 }
 
-export function WhaleSightingViewer({ loggedIn }: WhaleSightingViewerProps) {
-  const [selectedWhaleSpecies, setSelectedWhaleSpecies] = useState("");
-	const [selectedColour, setSelectedColour] = useState("");
-	const [selectedTailType, setSelectedTailType] = useState("");
-	const [selectedSize, setSelectedSize] = useState("");
-	const [minLat, setMinLat] = useState("");
-	const [maxLat, setMaxLat] = useState("");
-	const [minLog, setMinLog] = useState("");
-	const [maxLog, setMaxLog] = useState("");
-
-	function handleSearch(event: any) {
-		event.preventDefault();
+export function WhaleSightingViewer({loggedIn, isAdminPage} : WhaleSightingViewerProps) {
+
+	const [page, setPage] = useState(1);
+
+	const [sightings, setSightings] = useState<WhaleSighting[]>();
+
+	if (!isAdminPage) {
+		useEffect(() => {
+			fetchAllApprovedSightings()
+				.then(data => setSightings(data));
+		}, [])
 	}
-  
+	else {
+		useEffect(() => {
+			getPendingSightings()
+				.then(data => setSightings(data));
+		}, []);
+	}
+
+	if (!sightings) return <p>Waiting for data...</p>
+
 	return <>
+
 		<h2 className="whale-sighting-heading">Whale Sighting Viewer</h2>
 		<div className="whale-sighting-page">
-			<div className="whale-sighting-map-view-button">Switch to Map View</div>
-			<div className="whale-sighting-filter">
-				<WhaleSightingFilters
-				selectedWhaleSpecies={selectedWhaleSpecies} setSelectedWhaleSpecies={setSelectedWhaleSpecies}
-				selectedColour={selectedColour} setSelectedColour={setSelectedColour}
-				selectedTailType={selectedTailType} setSelectedTailType={setSelectedTailType}
-				selectedSize={selectedSize} setSelectedSize={setSelectedSize}
-				setMinLat={setMinLat} setMaxLat={setMaxLat} setMinLog={setMinLog} setMaxLog={setMaxLog} 
-				handleSearch={handleSearch} />
-			</div>
+			{isAdminPage ? <></> : <div className="whale-sighting-map-view-button">Switch to Map View</div>}
```

Comment:
We can write this another way using `&&` when we don't want to do something in the `false` case
```suggestion
			{!isAdminPage && <div className="whale-sighting-map-view-button">Switch to Map View</div>}
```